rosa_getting_started/rosa-deleting-cluster.adoc

```diff
@@ -7,4 +7,9 @@ toc::[]
 
 Delete a {product-title} (ROSA) cluster using the `rosa` command-line.
 
+[id="prerequisites_rosa-deleting-cluster"]
+== Prerequisites
+
+* If {product-title} created a VPC, you must remove network configurations, such as VPN configurations and VPC peering, and any additional services added to the VPC before you delete your cluster. If these configurations and services remain, the cluster does not delete properly.
```

Comment:
This is a little wordy for my taste. Maybe consider a bulleted list?

Reply:
"Delete" is a transitive verb. 

Could rewrite as something like: "If these configurations or services remain, the cluster cannot delete itself properly."

I'm also curious about what proper deletion entails. Will deletion fail predictably, and so users might follow a procedure elsewhere to fix things, is there chaos, will something error out somewhere, etc., but that's a whole can of worms. ;)